workflow.go

```diff
@@ -279,9 +284,15 @@ func PublishURLs(ttlHashSet *ttl_hash_set.TTLHashSet, queueManager *queue.Manage
 			continue
 		}
 
-		if crawlCount == AlreadyCrawled {
-			log.Debugln("URL extracted from page already crawled:", url)
-		} else if crawlCount == NotRecentlyCrawled {
+		// if crawlCount == AlreadyCrawled {
+			// log.Debugln("URL extracted from page already crawled:", url)
+		if crawlCount == Enqueued {
```

Comment:
Would be good to rename this to `crawlStatus` now, I don't think the count is relevant any more?

Reply:
it is still used as a count inside the actual crawler.